File: Python/Vehicle.py
class Vehicle:
    # Attribute Declaration
    __license_plate = "";
    __brand = "";
    __prod_year = -1;
    __color = "";

    # Constructor
    def __init__(self, license_plate="n/a", brand="n/a", prod_year=-1, color="n/a"):
        self.__license_plate = license_plate
        self.__brand = brand = brand
        self.__prod_year = prod_year
        self.__color = color

    # Getter-Setter
    # Get Methods
    def getLicense(self):
        return self.__license_plate
    def getBrand(self):
        return self.__brand
    def getProdYear(self):
        return self.__prod_year
    def getColor(self):
        return self.__color
        
    # Set Methods
    def setLicense(self, license_plate):
        self.__license_plate = license_plate
    def setBrand(self, brand):
        self.__brand = brand
    def setProdYear(self, prod_year):
        self.__prod_year = prod_year
    def setColor(self, color):
        self.__color = color
    

File: CPP/Vehicle.cpp
// Pre-processors and libraries
#pragma once
#include <string>

using namespace std;

// Vehicle Class
class Vehicle {
    private:
        // Vehicle Attributes
        string license_plate;
        string brand;
        int prod_year;
        string color;
    public:
        /* Constructors */

        // Empty Constructor
        Vehicle() {
            this->license_plate = -1;
            this->brand = "n/a";
            this->prod_year = -1;
            this->color = -1;
        }
        // Constructor
        Vehicle(string license_plate, string brand, int prod_year, string color){
            this->license_plate = license_plate;
            this->brand = brand;
            this->prod_year = prod_year;
            this->color = color;
        }

        /* Methods */
        // Get Methods
        string getLicense() { return this->license_plate; }
        string getBrand() { return this->brand; }
        int getProdYear() { return this->prod_year; }
        string getColor() { return this->color;}

        // Set Methods
        void setLicense(string license_plate) { this->license_plate = license_plate; }
        void setBrand(string brand) { this->brand = brand; }
        void setProdYear(int prod_year) { this->prod_year = prod_year; }
        void setColor(string color) { this->color = color; }
};